contracts/OwnedSet.sol

```diff
@@ -12,14 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// An owned validator set contract where the owner can add or remove validators.
+
 pragma solidity ^0.4.22;
 
 import "./Owned.sol";
 import "./ValidatorSet.sol";
 
 
-// Owner can add or remove validators.
-contract OwnedSet is Owned, ValidatorSet {
+contract BaseOwnedSet is Owned, ValidatorSet {
```

Comment:
Should we give some general structure to `contracts` directory?
Like:
```
`- contracts
   `- base/interfaces
   `- deployed (or just top-level)
   `- test
```
So that we have a clear view of what contracts should be deployed and which shouldn't?

This would imply splitting `OwnedSet.sol` into two files.